doc/api/errors.md

```diff
@@ -359,7 +359,7 @@ For example:
 
 ```js
 require('net').connect(-1);
-  // throws RangeError, port should be > 0 && < 65536
+  // throws RangeError, port should be >= 0 && < 65536
```

Comment:
The actual current error message is shown below, if you feel like changing it:

`RangeError: "port" option should be >= 0 and < 65536: -1`